knowing-more-about-ourselves.rst

```diff
@@ -0,0 +1,155 @@
+.. post:: September 12, 2022
+   :tags: metabase, data, grant
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+      Collect and analyze project and build data to make better decision about our product
+      with a strong focus on the user experience.
+
+Knowing more about how people use our service
+=============================================
+
+Read the Docs generates a lot of data.
+We are the largest documentation platform out there,
+with hundreds of thousands of projects using our product every day to host their documentation.
+This data includes simple things like number of users,
+builds using a particular Docker image,
+as well as more interesting ones like pageviews or Python dependencies installed via a ``requirements.txt`` file.
+We didn't collect this data in a systemic way during the first 10 years of our existence.
+
+Last year, with the growth of our product and the team,
+plus the `CZI grant we received <https://blog.readthedocs.com/czi-grant-announcement/>`_,
+we started asking ourselves some questions that we couldn't answer with the data we had.
+We decided to `start working on a project <https://github.com/readthedocs/readthedocs.org/pull/8124>`_
+to collect relevant data to answer a large number questions about how people use our service.
+
+What data are we collecting?
+----------------------------
+
+As you may already know, Read the Docs has a strict `privacy policy <https://docs.readthedocs.io/en/stable/privacy-policy.html>`_
+which we follow to ensure we are protecting the privacy of our users, authors and readers.
+It's still worth mentioning that most of the data we collect is publicly available,
+or is generated by our own service.
+
+The data generated by Read the Docs which we already had access to was mostly high-level about our database objects.
+A few examples would be:
+
+* project configuration (e.g. PR build enabled, default branch)
+* build metadata (e.g. number of builds, length of builds)
+* platform information (redirects, webhooks, search queries, pageviews)
+
+With our new data collections project,
+we were able to collect a lot of new, interesting data.
+This new data allows us to answer a whole new set of questions,
+and is mainly related to the build process
+The new information that we're collecting includes:
+
+* Python dependencies installed during a documentation build
+* Apt packages installed prior to building docs
+* The entire contents of the `Read the Docs configuration file <https://docs.readthedocs.io/en/stable/config-file/index.html>`_
```

Comment:
Now we are using a bullet list, we can add more things here if we want:

```suggestion
* OS and Python version used to build the documentation
* Python and Conda dependencies installed during a documentation build
* Apt packages installed prior to building docs
* The entire contents of the `Read the Docs configuration file <https://docs.readthedocs.io/en/stable/config-file/index.html>`_
```

I didn't include them original because it was just a paragraph and I they weren't too relevant. I'm fine if we skip them, tho... They aren't too relevant right at the moment.